CHANGELOG.rst

```diff
@@ -16,7 +16,11 @@ Unreleased
 
 *
 
-[0.4.2] - 2019-08-12
+[0.4.4] - 2019-08-13
+~~~~~~~~~~~~~~~~~~~~
+Add ability to filter by course grade, provided as a percentage to the endpoint.
+
+[0.4.3] - 2019-08-12
```

Comment:
I think this got out of sync with the actual version in `__init__.py`, by mistake, so I've correct it here to better reflect the state of things.